src/components/overlays/Tooltip/Tooltip.tsx

```diff
@@ -320,13 +323,18 @@ export const Tooltip = (props: TooltipProps) => {
 		useClickInsteadOfHover: !!useClickInsteadOfHover,
 	});
 
-	const isShowing = (forceHover || !stages.isStage0Hidden) && !hideTooltip;
+	const isShowing = (forceShow || !stages.isStage0Hidden) && !hideTooltip;
+	const isFirstRender = useRef(true);
 
 	useEffect(() => {
-		if (isShowing) {
-			onShow?.();
+		if (isFirstRender.current) {
```

Comment:
Curious about why you need this?

Reply:
On first render, `onHide()` is called without the check, just an annoying React quirk. See https://stackoverflow.com/questions/53253940/make-react-useeffect-hook-not-run-on-initial-render